calc.rb

```diff
@@ -0,0 +1,101 @@
+# Elsa's calculator project
+# Due: 150508
+# Breakfast + Lunch + Dinner
+
+# welcome message, instructions, and receiving user input
+print "\nWelcome to the Calculator!\n\nYou will be asked to enter\n  1) your first number,\n  2) your mathematical operation,\n  3) and your second number.\nThe resulting calculation will then be provided.\n"
```

Comment:
Good, you have instructions for the user. I would recommend breaking this line up a little so people reading your code don't have to scroll to see the entire line.